lib/ex_auction.ex

```diff
@@ -51,30 +51,34 @@ defmodule ExAuction do
           | {:error, ExAuction.Auction.Bid.Error.t()}
           | {:error, :auction_not_found}
           | {:error, :argument_error}
-  def place_bid(%Auction{} = auction, %Auction.Bid{user_id: user, value: value} = bid)
-      when is_binary(user) and is_integer(value) do
+  def place_bid(%Auction{} = auction, %Auction.Bid{value: value} = bid)
+      when is_integer(value) do
     ExAuction.Auction.Worker.bid(auction, bid)
   end
 
   def place_bid(_, _), do: {:error, :argument_error}
 
   @doc """
-  Start an auction.
+  Start a new auction or restart an existing one with optional bids list.
 
   ## Examples
 
-      iex> ExAuction.start(%ExAuction.Auction{})
+      iex> ExAuction.start(%ExAuction.Auction{}, bids \\ [])
       {:ok, %ExAuction.Auction{}}
 
   """
-  @spec start(ExAuction.Auction.t()) ::
+  @spec start(ExAuction.Auction.t(), [ExAuction.Auction.Bid.t()]) ::
```

Comment:
`@spec start(ExAuction.Auction.t(), [ExAuction.Auction.Bid.t() | _bids]) ::`

Reply:
What does this `| _bids` add to the spec 🤔